src/applications/appeals/995/components/EvidenceVaRecords.jsx

```diff
@@ -402,6 +410,7 @@ const EvidenceVaRecords = ({
               onPrimaryClick={handlers.onGoForward}
               onSecondaryClick={handlers.onGoBack}
               aria-describedby="nav-form-header"
+              uswds
```

Comment:
Does <Link> have `uswds` updates? I didn't see <Link> updated with uswds in the other files.

Reply:
This `uswds` is for the `VaButtonPair`. Expand the code to see.

Reply:
@Mottie Lol note to self: `Expand the code.`